rascal-vscode-extension/src/lsp/RascalLSPConnection.ts

```diff
@@ -72,172 +72,22 @@ export async function activateLanguageClient(
 
 async function showContentPanel(url: string): Promise<void> {
     // dispose of old panel in case it existed
+    const externalURL = (await vscode.env.asExternalUri(vscode.Uri.parse(url))).toString();
     const allOpenTabs = vscode.window.tabGroups.all.flatMap(tg => tg.tabs);
-    const tabsForThisPanel = allOpenTabs.filter(t => t.input instanceof vscode.TabInputWebview && t.label === url);
-    await vscode.window.tabGroups.close(tabsForThisPanel);
+    const tabsForThisPanel = allOpenTabs.filter(t => t.input instanceof vscode.TabInputWebview && t.label === externalURL);
+    
+    vscode.window.tabGroups.close(tabsForThisPanel);
```

Comment:
we should do an `await`  in front of this one (the code did this before as well, the diff is showing.

Reply:
o that would have been a nasty bug! thanks @DavyLandman